src/main/webapp/i18n/de/global.json

```
@@ -72,6 +72,10 @@
                 "metis": {
                     "post": "Beitrag",
                     "answerPost": "Antwort"
+                },
+                "messages": {
+                    "conversation": "Gespräch",
+                    "conversationParticipant": "Unterhaltungsdetails"
```

```suggestion
                    "conversationParticipant": "Konversationsteilehmer"
```
or maybe "Teilnehmer der Konversation"

---

removed this part as it is no longer needed